app/views/posts/index.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>Posts</h1>
-<p>Here is a list of posts for a given user</p>
+<main>
+<section class="posts">
+  <img class="image" src="<%=@user.photo%>" alt="user_image" />
+  <div class="details">
+    <h2><%= @user.name%> </h2>
+    <p>Number of posts: <%= @user.posts_counter%> </p>
+  </div> 
+</section>
+
+<section class="usercontainer">
+  <% @user.posts.each do |post| %>
+    <%= render partial: 'post', locals: {post: post} %>
+  <% end %>
+</section>
+</main>
```

Comment:
- [ ] Please show the post number on this page as it is available on the wireframe `post #1`. Your view is missing that.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/175052660-c6141e68-639a-4975-bd20-d9d8e83257ce.png)

Reply:
@thecodechaser Thanks for the review, issue resolved.